### PR TITLE
Feature/leichte sprache

### PR DESCRIPTION
**Pull request by tcfos**
## Checklist

* [x] This PR does not change text that hasn't been signed off with the RKI. Have a look at the pinned issue [440](https://github.com/corona-warn-app/cwa-app-ios/issues)
* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.

## Description

This solves https://jira.itc.sap.com/browse/EXPOSUREAPP-1988

Implements a link to https://www.bundesregierung.de/breg-de/themen/corona-warn-app/corona-warn-app-leichte-sprache-gebaerdensprache ("Informationen zur App in leichter Sprache und Gebärdensprache") on 
* the onboarding screen
* on the "About" screen
The links will be displayed only in case the iPhone Language has been set to "de" (German).

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 1]
```diff
@@ -317,6 +317,8 @@ enum AppStrings {
 		static let onboardingInfo_togetherAgainstCoronaPage_title = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_title", comment: "")
 		static let onboardingInfo_togetherAgainstCoronaPage_boldText = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_boldText", comment: "")
 		static let onboardingInfo_togetherAgainstCoronaPage_normalText = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_normalText", comment: "")
+		static let onboardingInfo_togetherAgainstCoronaPage_Link = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_Link", comment: "URL")
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:320]:
```suggestion
		static let onboardingInfo_togetherAgainstCoronaPage_link = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_Link", comment: "URL")
```

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift [hunk 2]
```diff
@@ -167,6 +168,24 @@ final class OnboardingInfoViewController: UIViewController {
 		textLabel.text = onboardingInfo.text
 		textLabel.isHidden = onboardingInfo.text.isEmpty
 
+		let language = Bundle.main.preferredLocalizations.first ?? ""
+		if language == "de" {
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:172]: Similar to above:
```suggestion
              if Bundle.main.preferredLocalizations.first == "de" {
```

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 3]
```diff
@@ -317,6 +317,8 @@ enum AppStrings {
...
+		static let OnboardingInfo_togetherAgainstCoronaPage_LinkText = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_LinkText", comment: "")
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:321]:
```suggestion
		static let onboardingInfo_togetherAgainstCoronaPage_linkText = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_LinkText", comment: "")
```

**Review by whiskey — APPROVED** [review]
Approved from my side. Feel free to consider the suggestions or discard them :)

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift (hunk 1 above)
Comment (haosap) [reply src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:323]: I even suggest the camel format for our project.

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift (hunk 3 above)
Comment (tcfos) [reply src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift]: Included this suggestion.

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift (hunk 1 above)
Comment (tcfos) [reply src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift]: included this suggestion.

File: src/xcode/ENA/ENA.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "FMDB",
```
Comment (MichaelPFrey) [inline src/xcode/ENA/ENA.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:5]: why does this pull request add new packages?

Comment (tcfos) [reply src/xcode/ENA/ENA.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:5]: no new dependencies. We checked in the Package.resolved file

**Review by ArturFriesen — APPROVED** [review]

**Review by janetback — COMMENTED** [review]
UA Reviewd

**Pull request merged**